src/test/java/core/basesyntax/service/implementations/WriteServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service.implementations;
+
+import core.basesyntax.service.inerfaces.WriteService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriteServiceImplTest {
+    private static final String TO_FILE_ROUTE
+            = "src/test/java/core/basesyntax/resources/ToFile.cvs";

```

Comment:
```suggestion
            = "src/test/resources/ToFile.cvs";
```

Reply:
java.lang.RuntimeException: Can't write to file src/test/resources/ToFile.cvs